server/lib/flow-event.js

```diff
@@ -129,19 +129,20 @@ module.exports = (req, metrics, requestReceivedTime) => {
   const navigationTiming = metrics.navigationTiming;
   if (emitPerformanceEvents && navigationTiming) {
     PERFORMANCE_TIMINGS.forEach(item => {
-      const time = item.timings.reduce((sum, timing) => {
+      const relativeTime = item.timings.reduce((sum, timing) => {
         const from = navigationTiming[timing.from];
         const until = navigationTiming[timing.until];
         if (from >= 0 && until > from) {
           sum += until - from;
         }
         return sum;
       }, 0);
+      const absoluteTime = metrics.flowBeginTime + relativeTime;
 
-      if (time > 0) {
+      if (relativeTime > 0 && isValidTime(absoluteTime, requestReceivedTime)) {
         logFlowEvent({
-          flowTime: time,
-          time: metrics.flowBeginTime + time,
+          flowTime: relativeTime,
+          time: absoluteTime,
           type: `flow.performance.${item.event}`
         }, metrics, req);
       }
```

Comment:
SGTM, I'm going to open an issue to discuss whether we should log data that's dropped on the ground. @jrgm asked for it in [@vladikoff's Sentry PR](https://github.com/mozilla/fxa-content-server/pull/4840#discussion_r107500278) so I want to make sure we are doing the right thing and folks are happy.